content/cli/_index.md

```diff
@@ -69,21 +77,18 @@ After obtaining the binary, run it from the terminal with any of the available c
 
 ## Available commands
 
-|Command                                       |Description                                |
-|:---------------------------------------------|:------------------------------------------|
-|**Start**                                     |                                           |
-|[`init`]({{< relref "/cli/kes-init" >}})          |Initialize a stateful KES server or cluster|
-|[`server`]({{< relref "/cli/kes-server" >}})      |Start a KES server                         |
-|**Manage**                                    |                                           |
-|[`enclave`]({{< relref "/cli/kes-enclave" >}})    |Manage KES enclaves                        |
-|[`identity`]({{< relref "/cli/kes-identity" >}})  |Manage KES identities                      |
-|[`key`]({{< relref "/cli/kes-key" >}})            |Manage cryptographic keys                  |
-|[`policy`]({{< relref "/cli/kes-policy" >}})      |Manage KES policies                        |
-|[`secret`]({{< relref "/cli/kes-secret" >}})      |Manage KES secrets                         |
-|**Audit**                                     |                                           |
-|[`log`]({{< relref "/cli/kes-log" >}})            |Print error and audit log events           |
-|[`metric`]({{< relref "/cli/kes-metric" >}})      |Print server metrics                       |
-|[`status`]({{< relref "/cli/kes-status" >}})      |Print server status                        |
-|**Administer**                                |                                           |
-|[`migrate`]({{< relref "/cli/kes-migrate" >}})    |Migrate KMS data                           |
-|[`update`]({{< relref "/cli/kes-update" >}})      |Update KES binary                          |
\ No newline at end of file
+|Command                                           |Description                            |
+|:-------------------------------------------------|:--------------------------------------|
```

Comment:
@aead is it worth maintaining something like a deprecated API list for users on older KES versions?

We do something similar for the Server/MC docs where we keep deprecated commands + the version we dropped them from. It slightly simplifies support for users who are pinned to the older version.

cc @djwfyi

Reply:
At a glance, we are dropping:

`enclave`
`manage`
`init`